test/unit/acl/access-controller.test.ts

```diff
@@ -7,6 +7,10 @@ import { solidLogicSingleton } from '../../../src/logic'
 const store = solidLogicSingleton.store
 
 silenceDebugMessages()
+jest.mock('solid-auth-client', () => ({
+  currentSession: () => Promise.resolve(),
+  trackSession: () => null
```

Comment:
`trackSession` is called from solid-logic now, so it has to be a function. I also added `currentSession`, to remove some error messages from the logs (which did not break the test, however)